portingdb/htmlreport.py

```diff
@@ -269,13 +269,13 @@ def format_time_ago(date):
         else:
             return "a year ago"
     # Months
-    elif diff.days >= 31:
+    elif diff.days >= 8*7:
         if diff.days >= 2 * 30:
             return "{} months ago".format(math.floor(diff.days / 30))
         else:
             return "a month ago"
```

Comment:
the first elif means >= 56 days, the second one is >= 60. There is an else for "a month ago" for 4 days only. Is that worth it? It's almost 2 months anyway at that point. If we change `8*7` to `9*7`, we can get rid of the inner if.